tests/test_order.py

```diff
@@ -515,3 +517,70 @@ def test_add_order_note_view(order, authorized_client, customer_user):
     assert get_redirect_location(response) == redirect_url
     order.refresh_from_db()
     assert order.customer_note == customer_note
+
+
+def _calculate_order_weight_from_lines(order):
+    weight = Weight(kg=0)
+    for line in order:
+        weight += line.variant.get_weight() * line.quantity
+    return weight
+
+
+def test_order_weight(order_with_lines):
+    order_weight = order_with_lines.weight
+    calculated_weight = _calculate_order_weight_from_lines(order_with_lines)
+
+    assert calculated_weight == order_weight
+
+
+def test_order_weight_add_more_variant(order_with_lines):
+    variant = order_with_lines.lines.first().variant
+
+    add_variant_to_order(order_with_lines, variant, 2)
+    order_with_lines.refresh_from_db()
+
+    assert order_with_lines.weight == _calculate_order_weight_from_lines(
+        order_with_lines)
+
+
+def test_order_weight_add_new_variant(order_with_lines, product):
+    variant = product.variants.first()
+
+    add_variant_to_order(order_with_lines, variant, 2)
+    order_with_lines.refresh_from_db()
+
+    assert order_with_lines.weight == _calculate_order_weight_from_lines(
+        order_with_lines)
+
+
+def test_order_weight_change_line_quantity(order_with_lines):
+    line = order_with_lines.lines.first()
+    new_quantity = line.quantity + 2
+    change_order_line_quantity(line, new_quantity)
+    order_with_lines.refresh_from_db()
+
+    assert order_with_lines.weight == _calculate_order_weight_from_lines(
+        order_with_lines)
+
+
+def test_order_weight_delete_line(order_with_lines):
+    line = order_with_lines.lines.first()
+    delete_order_line(line)
+
+    assert order_with_lines.weight == _calculate_order_weight_from_lines(
+        order_with_lines)
+
+
+def test_get_order_weight_non_existing_product(order_with_lines, product):
+    # Removing product should not affect order's weight
```

Comment:
```suggestion
    """Removing product should not affect the order's weight."""
```

Reply:
In tests, we rather use line comments like this; what you suggest is more suitable for docstrings.